packages/clay-css/src/scss/components/_buttons.scss

```diff
@@ -1,3 +1,32 @@
+// Remove the default `border-radius` that macOS Chrome adds.
```

Comment:
I'm worried about the legality of such changes... we're doing:

- Copy paste `_reboot.scss` which is in fact `Normalize.css` keeping its LICENSE
- Strip our copy for parts and remove all traces of its LICENSE

While both `Bootstrap` and `Normalize.css` are MIT, and of course ianal, this doesn't feel right.

I was initially going to comment just on the first commit and ask why we needed to "absorb" this rather than just adding a direct dependency on `Normalize.css`, then this commit further pushed my concerns...

I know @wincent is partial (me too) to having no-dependencies, specially when they don't pull their own weight, so I'm curious to hear his opinion (and everyone else's) on this...

Reply:
You should never ask me to talk about the costs and perils of dependencies... I may go on for several hours!

On the subject of the licensing question, IANALE (I am not a lawyer either), but the MIT is quite clear about the need to preserve the license text in any derivative work.

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

The "substantial" bit is where an expert opinion could weigh in — and of course, whatever that opinion might be, I bet you could find another expert to offer an equally hefty counter-opinion 😂. If in doubt, I just keep the original license.

You might argue that if we fork something hard and then change every single thing about it, then it is no longer a derivative of the original work (see: [https://en.wikipedia.org/wiki/Ship_of_Theseus](Ship of Theseus), but I tend to err on the side of caution. A "clean room" dependency-free from-scratch reset would obviously liberate of this question, but I don't know if we'd bother with that. `Normalize.css` seems bloat-free enough that it's not really going to swell our dependency footprint by much.

Reply:
I didn't think about attribution too much. I'm good with using normalize or reboot. I do think Bootstrap's _reboot.scss would be easier since it's already in use.